### PR TITLE
Create an issue template for gathering details of bugs

### PR DESCRIPTION
**Pull request by muthuswamyopn**
Use this issue template when filing a bug. The template helps Opn gather needed bug details to resolve issues faster.

### CODE REVIEW

File: ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md:23]: What's the purpose of this blank line?

File: ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Logs/Error messages
+
+_Attach or paste appropriate logs and error messages to help us debug better._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_Attach or paste appropriate logs and error messages to help us debug more easily._
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Name and Version Information
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
## Name and version information
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+
+## Logs/Error messages
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
## Logs/error messages
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+_Attach or paste appropriate logs and error messages to help us debug better._
+
+## Name and Version Information
+
+_Specify the name and version of the following components:_
+
+_* Development Environment (For example, XCode version 12.0)._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_* Development Environment (for example, XCode version 12.0)._
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+_* SDK (For example, Omise-iOS version 4.25)._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_* SDK (for example, Omise-iOS version 4.25)._
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+_* Package Manager (For example, SwiftPM version 0.6.0)._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_* Package Manager (for example, SwiftPM version 0.6.0)._
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+_* Operating System (For example, iOS version 17.1.1)._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_* Operating System (for example, iOS version 17.1.1)._
```

File: ISSUE_TEMPLATE/bug-report.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+_* Browser, if it is a web issue (For example, Firefox version 119.0.1)._
```
Comment (danfowler) [inline ISSUE_TEMPLATE/bug-report.md]:
```suggestion
_* Browser, if it is a web issue (for example, Firefox version 119.0.1)._
```

File: ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (muthuswamyopn) [reply ISSUE_TEMPLATE/bug-report.md:23]: I removed it.

**Comment by danfowler** [comment]
@muthuswamyopn Can you please make sure you provide a clear PR title and description?

**Comment by muthuswamyopn** [comment]
> @muthuswamyopn Can you please make sure you provide a clear PR title and description?
Added @danfowler

**Review by danfowler — APPROVED** [review]

**Pull request merged**